src/Microsoft.CodeAnalysis.BannedApiAnalyzers/UnitTests/SymbolIsBannedAnalyzerTests.cs

```diff
@@ -761,6 +762,147 @@ void M()
                 GetCSharpResultAt(10, 31, SymbolIsBannedAnalyzer.SymbolIsBannedRule, "C.Banned()", ""));
         }
 
+        [Fact, WorkItem(3295, "https://github.com/dotnet/roslyn-analyzers/issues/3295")]
+        public async Task CSharp_BannedAbstractVirtualMemberAlsoBansOverrides()
+        {
+            var source = @"
+using System;
+
+namespace N
+{
+    public abstract class C1
+    {
+        public abstract void Method1();
+        public abstract int Property1 { get; set; }
+        public abstract event Action Event1;
+
+        public virtual void Method2() {}
+        public virtual int Property2 { get; set; }
+        public virtual event Action Event2;
+    }
+
+    public class C2 : C1
+    {
+        public override void Method1() {}
+        public override int Property1 { get; set; }
+        public override event Action Event1;
+
+        public override void Method2()
+        {
+            base.Method2();
```

Comment:
The current implementation raises a diagnostic on this line but I don't think we should. I think that when an API is banned we either want to prevent all usages, in which case just the `override` definition itself should trigger the diagnostic or we want to prevent calls but we do allow overrides in which case the call to base in the overrides should be fine.

Reply:
Can we use the inline annotations in tests for diagnostics? It is very difficult to map line numbers to which code actually fired the diagnostic.

Reply:
I will need to duplicate the code but yeah sure!